.travis.yml

```diff
@@ -7,16 +8,17 @@ cache:
   yarn: true
 services:
   - redis-server
-  - mysql
+  - postgresql
+addons:
+  - postgresql: "13"
```

Comment:
https://docs.travis-ci.com/user/database-setup/#using-a-different-postgresql-version mentions the need for extra config in the case of v11 and v12, but doesn't say anything about v13.

Reply:
Yeah the fact that it worked seemed good enough for me.